pkg/ingress/model_builder.go

```diff
@@ -280,7 +280,7 @@ func (t *defaultModelBuildTask) mergeListenPortConfigs(_ context.Context, listen
 		inboundCIDRv4s: mergedInboundCIDRv4s.List(),
 		inboundCIDRv6s: mergedInboundCIDRv6s.List(),
 		sslPolicy:      mergedSSLPolicy,
-		tlsCerts:       mergedTLSCerts.List(),
+		tlsCerts:       mergedTLSCerts.UnsortedList(),
```

Comment:
This will list out the certs in random order and the outcome is not deterministic. We'd need to consider the following issues
- maintain the order specified in the annotation
- handle multiple ingresses with the certificate annotation (provide some reasonable ordering)

Reply:
> maintain the order specified in the annotation

I have two ideas.

1 
    Create a field called defaultCertificate in the request struct for the v1 implementation.
    https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/9776d298fddc1bb7fe9245510467d2507580ec96/internal/alb/ls/listener.go#L241-L246
2. 
    Stop using sets in mergedTLSCerts and change to slice.
    
> handle multiple ingresses with the certificate annotation (provide some reasonable ordering)

I don't have an idea, but I think it's going to be a big change because I can't go without changing the specifications below.

https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/9776d298fddc1bb7fe9245510467d2507580ec96/docs/guide/ingress/annotation.md#certificate-arn

However, I have the same problem as the issue below, so I would like a solution.
https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/1836

Reply:
Alternatively, evaluate in the order of group order, and make group order the largest one first.
https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/48b4a7dd3177cab816dd678cdec460550e0cd528/docs/guide/ingress/annotations.md#group.order